openfaas/config.yaml

```diff
@@ -0,0 +1,64 @@
+# this will be replaced by Okteto during deployment
+functionNamespace: $NAMESPACE
+serviceType: ClusterIP
+
+# okteto manages the RBAC 
+rbac: false
+
+# disabled by default to make development easier
+basic_auth: false
+
+gateway:
+  resources:
+    limits:
+      memory: "120Mi"
+      cpu: "50m"
+
+faasnetes:
+  resources:
+    limits:
+      memory: "120Mi"
+      cpu: "50m"
+
+queueWorker:
+    limits:
+      memory: "120Mi"
+      cpu: "50m"
+
+prometheus:
+  resources:
+    limits:
+      memory: "512Mi"
+      cpu: "100m"
+
+alertmanager:
+    limits:
+      memory: "50Mi"
+      cpu: "100m"
+
+nats:
+  resources:
+    limits:
+      memory: "120Mi"
+    requests:
```

Comment:
Set just the `limits`, Okteto Cloud sets `requests` to zero.
Also, setting `limits.cpu` we can reduce the total cpu needed